modelscope/models/cv/image_view_transform/ldm/autoencoder.py

```diff
@@ -3,7 +3,7 @@
 import pytorch_lightning as pl
 import torch
 import torch.nn.functional as F
-from taming.modules.vqvae.quantize import VectorQuantizer
+from taming.modules.vqvae.quantize import VectorQuantizer2 as VectorQuantizer
```

Comment:
similar to the comment above, why this change?

also how come introducing new audio models will include change in cv modules?